### PR TITLE
First stab at the feature

### PR DESCRIPTION
**Pull request by githubteacher**

### CODE REVIEW

**Comment by githubteacher** [comment]
@kakuls Check this out.

**Comment by kakuls** [comment]
This is a good start. Thanks for giving me a sneak peak early

**Comment by githubteacher** [comment]
This closes #4 when fully implemented

File: sample.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Lorem ipsum dolor sit amet, vitae risus eu. Risus pede. Etiam facilisi quis, iaculis cum sed, eu mauris. Magna turpis. Etiam sed voluptatem.
+
+In nam auctor. Dictum imperdiet commodo, lacinia urna, tortor mi. Enim leo urna, justo felis, adipiscing erat. Duis nulla. Semper leo, mi ut rutrum. Ut congue ac.
```
Comment (githubteacher) [inline sample.js]: @harsh Any particular thoughts on this?

Comment (harsh) [inline sample.js]: @matthewmccullough looks great

**Pull request merged**